fairroot/online/source/FairLmdSource.h

```diff
@@ -65,6 +65,9 @@ class FairLmdSource : public FairMbsSource
 
     FairLmdSource& operator=(const FairLmdSource&);
 
+  private:
+    void CloseLmd();
```

Comment:
This should be an overridden private virtual function.

Reply:
Let's please postpone this for another PR as well.